librz/analysis/meta.c

```diff
@@ -237,8 +237,8 @@ RZ_API const char *rz_meta_type_to_string(int type) {
 	return "# unknown meta # ";
 }
 
-RZ_API void rz_meta_print(RzAnalysis *a, RzAnalysisMetaItem *d, ut64 start, ut64 size, int rad, PJ *pj, bool show_full) {
-	rz_return_if_fail(!(rad == 'j' && !pj)); // rad == 'j' => pj != NULL
+RZ_API void rz_meta_print(RzAnalysis *a, RzAnalysisMetaItem *d, ut64 start, ut64 size, RzOutputMode rad, PJ *pj, bool show_full) {
```

Comment:
I suggest to rename variable too `rad` -> `mode`. And everywhere else you meet unclear name for the mode.

Reply:
Ok, will do.

Reply:
I have changed `rad` to `mode` wherever output modes are involved.